src/pages/workspace/categories/ImportedCategoriesPage.tsx

```diff
@@ -0,0 +1,171 @@
+import {useOnyx} from 'react-native-onyx';
+import ConfirmModal from '@components/ConfirmModal';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import type {ColumnRole} from '@components/ImportColumn';
+import ImportSpreadsheetColumns from '@components/ImportSpreadsheetColumns';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useLocalize from '@hooks/useLocalize';
+import usePolicy from '@hooks/usePolicy';
+import {closeImportPage} from '@libs/actions/ImportSpreadsheet';
+import {importPolicyCategories} from '@libs/actions/Policy/Category';
+import Navigation from '@libs/Navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@libs/Navigation/types';
+import {isControlPolicy} from '@libs/PolicyUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+
+type ImportedCategoriesPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.CATEGORIES_IMPORTED>;
+
+function numberToColumn(index: number) {
+    let column = '';
+    let number = index;
+    while (number >= 0) {
+        column = String.fromCharCode((number % 26) + 65) + column;
+        number = Math.floor(number / 26) - 1;
+    }
+    return column;
+}
+
+function generateColumnNames(length: number) {
+    return Array.from({length}, (_, i) => numberToColumn(i));
+}
+
+function findDuplicate(array: string[]): string | null {
+    const frequencyCounter: Record<string, number> = {};
+
+    for (const item of array) {
+        if (item !== CONST.CSV_IMPORT_COLUMNS.IGNORE) {
+            if (frequencyCounter[item]) {
+                return item;
+            }
+            frequencyCounter[item] = (frequencyCounter[item] || 0) + 1;
+        }
+    }
+
+    return null;
+}
+
+function ImportedCategoriesPage({route}: ImportedCategoriesPageProps) {
+    const {translate} = useLocalize();
+    const [spreadsheet] = useOnyx(ONYXKEYS.IMPORTED_SPREADSHEET);
+    const [isImportingCategories, setIsImportingCategories] = useState(false);
+    const [containsHeader, setContainsHeader] = useState(false);
+    const policyID = route.params.policyID;
+    const policy = usePolicy(policyID);
+    const columnNames = generateColumnNames(spreadsheet?.data?.length ?? 0);
+
+    const isControl = isControlPolicy(policy);
+
+    const getColumnRoles = (): ColumnRole[] => {
+        const roles = [];
+        roles.push(
+            {text: translate('common.ignore'), value: CONST.CSV_IMPORT_COLUMNS.IGNORE},
+            {text: translate('common.name'), value: CONST.CSV_IMPORT_COLUMNS.NAME, isRequired: true},
+            {text: translate('common.enabled'), value: CONST.CSV_IMPORT_COLUMNS.ENABLED, isRequired: true},
+        );
+
+        if (isControl) {
+            roles.push({text: translate('workspace.categories.glCode'), value: CONST.CSV_IMPORT_COLUMNS.GL_CODE, isRequired: true});
```

Comment:
GL codes are not required
```suggestion
            roles.push({text: translate('workspace.categories.glCode'), value: CONST.CSV_IMPORT_COLUMNS.GL_CODE, isRequired: false});
```